packages/babel-parser/src/util/whitespace.ts

```diff
@@ -28,7 +28,7 @@ export function hasNewLine(input: string, start: number, end: number): boolean {
   return false;
 }
 
-export const skipWhiteSpace = /(?:\s|\/\/.*|\/\*[^]*?\*\/)*/g;
+export const skipWhiteSpace = /(?:\s|\/\/.*|\/\*[\s\S]*?\*\/)*/g;
```

Comment:
`^` is shorter than `\s\S`, aren't they equivalent here?

Reply:
Maybe we can override the `allows` config of `regexp/match-any`:
```json
{
  "regexp/match-any": ["error", {
    "allows": ["[^]"]
  }]
}
```
 https://ota-meshi.github.io/eslint-plugin-regexp/rules/match-any.html#allows